### PR TITLE
Bioinfo sample

### PR DESCRIPTION
**Pull request by mayabrandi**

### CODE REVIEW

File: vogue/adapter/plugin.py [hunk 1]
```diff
@@ -198,6 +198,32 @@ def add_or_update_analysis_bioinfo_processed(self, analysis_result: dict):
                 upsert=True)
             LOG.info("Updated analysis for sample %s.", case_id)
 
+    def add_or_update_analysis_bioinfo_samples(self, analysis_result: dict):
```
Comment (mayabrandi) [inline vogue/adapter/plugin.py]:
```suggestion
    def add_or_update_bioinfo_samples(self, analysis_result: dict):
```

File: vogue/adapter/plugin.py [hunk 2]
```diff
@@ -198,6 +198,32 @@ def add_or_update_analysis_bioinfo_processed(self, analysis_result: dict):
...
+        """Functionality to add or update analysis for sample level results"""
+        print(analysis_result.keys())
```
Comment (mayabrandi) [inline vogue/adapter/plugin.py]:
```suggestion
```

File: vogue/build/case_analysis.py [hunk 3]
```diff
@@ -265,3 +259,36 @@ def build_analysis(analysis_dict: dict,
                                           processed=process_case)
 
     return mongo_doc
+
+def build_analysis_sample(analysis_dict: dict,  sample_id: str,process_case=False):
```
Comment (mayabrandi) [inline vogue/build/case_analysis.py]:
```suggestion
def build_bioinfo_sample(analysis_dict: dict,  sample_id: str,process_case=False):
```

File: vogue/commands/load/bioinfo/bioinfo_process.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    is_flag=True,
+    help=
+    'Specify this flag if input json should be cleanup based on analysis-type and models.'
+)
+@click.option(
+    '--case-analysis-type',
+    type=click.Choice(['multiqc', 'microsalt', 'custom']),
+    default='multiqc',
+    help=
+    'Specify the type for the case analysis. i.e. if it is multiqc output, then choose multiqc'
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply vogue/commands/load/bioinfo/bioinfo_process.py]: Analysis type is to filter out which portion of model to load into db. This is developer only, and  shouldn't be used by user. I'll fix the help text.

File: vogue/commands/load/bioinfo/bioinfo_process.py [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+              required=True,
+              help='Analysis workflow used.')
+@click.option('--workflow-version',
+              required=True,
+              help='Analysis workflow used.')
+@click.option(
+    '--cleanup/--not-cleanup',
+    is_flag=True,
+    help=
+    'Specify this flag if input json should be cleanup based on analysis-type and models.'
```
[the remark this answers is not in the document]
Comment (hassanfa) [reply vogue/commands/load/bioinfo/bioinfo_process.py]: microsalt :-)

**Comment by hassanfa** [comment]
I think this is in good condition, lets merge it.

**Comment by mayabrandi** [comment]
Its getting cleaner!

I think we can merge now 👯‍♂

**Comment by hassanfa** [comment]
Cool! Merging it.

**Pull request merged**